dubbo mod build failed

### Ⅰ. Issue Description

when i try to build by following https://github.com/alibaba/tengine/tree/master/modules/mod_dubbo

### Ⅱ. Describe what happened
Sth wroung.
```
cc -c -pipe  -O -W -Wall -Wpointer-arith -Wno-unused-parameter -Werror -g   -I src/core -I src/event -I src/event/modules -I src/os/unix -I src/proc -I ./pcre-8.43/ -I ./openssl-1.0.2s//.openssl/include -I ./zlib-1.2.11 -I objs -I src/http -I src/http/modules -I ./modules/mod_dubbo -I ./modules/mod_dubbo/utils -I ./modules/mod_dubbo/hessian2 -I ./modules/ngx_multi_upstream_module \
        -o objs/addon/mod_dubbo/ngx_dubbo_util.o \
        ./modules/mod_dubbo/ngx_dubbo_util.cpp
./modules/mod_dubbo/ngx_dubbo_util.cpp: In function ‘ngx_int_t ngx_dubbo_hessian2_encode_str(ngx_pool_t*, ngx_str_t*, ngx_str_t*)’:
./modules/mod_dubbo/ngx_dubbo_util.cpp:43:27: error: catching polymorphic type ‘class hessian::io_exception’ by value [-Werror=catch-value=]
   43 |     } catch (io_exception e) {
      |                           ^
./modules/mod_dubbo/ngx_dubbo_util.cpp: In function ‘ngx_int_t ngx_dubbo_hessian2_encode_map(ngx_pool_t*, ngx_array_t*, ngx_str_t*)’:
./modules/mod_dubbo/ngx_dubbo_util.cpp:77:27: error: catching polymorphic type ‘class hessian::io_exception’ by value [-Werror=catch-value=]
   77 |     } catch (io_exception e) {
      |                           ^
./modules/mod_dubbo/ngx_dubbo_util.cpp: In function ‘ngx_int_t ngx_dubbo_hessian2_encode_payload_map(ngx_pool_t*, ngx_array_t*, ngx_str_t*)’:
./modules/mod_dubbo/ngx_dubbo_util.cpp:119:27: error: catching polymorphic type ‘class hessian::io_exception’ by value [-Werror=catch-value=]
  119 |     } catch (io_exception e) {
      |                           ^
./modules/mod_dubbo/ngx_dubbo_util.cpp: In function ‘ngx_int_t ngx_dubbo_hessian2_decode_payload_map(ngx_pool_t*, ngx_str_t*, ngx_array_t**, ngx_log_t*)’:
./modules/mod_dubbo/ngx_dubbo_util.cpp:206:27: error: catching polymorphic type ‘class hessian::io_exception’ by value [-Werror=catch-value=]
  206 |     } catch (io_exception e) {
      |                           ^
cc1plus: all warnings being treated as errors
make[1]: *** [objs/Makefile:1341: objs/addon/mod_dubbo/ngx_dubbo_util.o] Error 1
make[1]: Leaving directory '/home/mark4z/Downloads/tengine'
make: *** [Makefile:8: build] Error 2
```

### Ⅲ. Describe what you expected to happen
build sucessful.

### Ⅳ. How to reproduce it (as minimally and precisely as possible)

1.just follow https://github.com/alibaba/tengine/tree/master/modules/mod_dubbo
2.
3.



### Ⅴ. Anything else we need to know?

1. If applicable, add nginx  [debug log doc](http://nginx.org/en/docs/debugging_log.html).
2.
3.

### Ⅵ. Environment:

- Tengine version (use `sbin/nginx -V`):
- OS (e.g. from /etc/os-release): Ubuntu 20.04.2 LTS
- Kernel (e.g. `uname -a`):Linux nkgw10nh3481 4.4.0-17763-Microsoft #379-Microsoft Wed Mar 06 19:16:00 PST 2019 x86_64 x86_64 x86_64 GNU/Linux
- Others:

--- a/modules/mod_dubbo/ngx_dubbo_util.cpp
+++ b/modules/mod_dubbo/ngx_dubbo_util.cpp
@@ -40,7 +40,7 @@ ngx_int_t ngx_dubbo_hessian2_encode_str(ngx_pool_t *pool, ngx_str_t *in, ngx_str
         out->len = str.length();
 
         return NGX_OK;
-    } catch (io_exception e) {
+    } catch (io_exception &e) {
         ngx_log_error(NGX_LOG_ERR, pool->log, 0, "dubbo: parse exception failed %s", e.what());
         return NGX_ERROR;
     } catch (...) {
@@ -74,7 +74,7 @@ ngx_int_t ngx_dubbo_hessian2_encode_map(ngx_pool_t *pool, ngx_array_t *in, ngx_s
         out->len = str.length();
 
         return NGX_OK;
-    } catch (io_exception e) {
+    } catch (io_exception &e) {
         ngx_log_error(NGX_LOG_ERR, pool->log, 0, "dubbo: parse exception failed %s", e.what());
         return NGX_ERROR;
     } catch (...) {
@@ -116,7 +116,7 @@ ngx_int_t ngx_dubbo_hessian2_encode_payload_map(ngx_pool_t *pool, ngx_array_t *i
         out->len = str.length();
 
         return NGX_OK;
-    } catch (io_exception e) {
+    } catch (io_exception &e) {
         ngx_log_error(NGX_LOG_ERR, pool->log, 0, "dubbo: parse exception failed %s", e.what());
         return NGX_ERROR;
     } catch (...) {
@@ -203,7 +203,7 @@ ngx_dubbo_hessian2_decode_payload_map(ngx_pool_t *pool, ngx_str_t *in, ngx_array
         }
 
         return NGX_OK;
-    } catch (io_exception e) {
+    } catch (io_exception &e) {
         ngx_log_error(NGX_LOG_ERR, log, 0, "dubbo: parse exception failed %s", e.what());
         return NGX_ERROR;
     } catch (...) {

